.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTORY_RELEASE_REPO: eaiw-release-local
+      ARTIFACTORY_RESOLVE_REPO: eaiw-local
+      ARTIFACTORY_SNAPSHOT_REPO: eaiw-snapshot-local
+      ARTIFACTORY_CONTEXT: ${{ secrets.ARTIFACTORY_CONTEXT }}
+      ARTIFACTORY_PUBLISH_USER: ${{ secrets.ARTIFACTORY_PUBLISH_USER }}
+      ARTIFACTORY_PUBLISH_PASSWORD: ${{ secrets.ARTIFACTORY_PUBLISH_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
This settings makes it fetch the full history of repo. It is needed so that gradle-gitflow can infer the version of the artifact to produce.